sw/device/tests/crypto/cryptotest/firmware/sca_lib.h

```diff
@@ -5,8 +5,36 @@
 #ifndef OPENTITAN_SW_DEVICE_TESTS_CRYPTO_CRYPTOTEST_FIRMWARE_SCA_LIB_H_
 #define OPENTITAN_SW_DEVICE_TESTS_CRYPTO_CRYPTOTEST_FIRMWARE_SCA_LIB_H_
 
```

Comment:
I needed to add
#include <stdint.h>

Reply:
Ah thanks, done.
`#include "sw/device/lib/ujson/ujson.h"`
was missing.